tests/test_validation_tool.py

```diff
@@ -179,3 +192,98 @@ def test_abort_on_error() -> None:
     )
     assert isinstance(ret, str)
     assert re.match(r"^(tests/data/ethernet/invalid/).+(\.raw) (classified as FalseNegative)$", ret)
+
+
+def test_cli_not_regular_file(tmpdir: Path) -> None:
+    subdir = tmpdir / "test"
+    subdir.mkdir()
+    assert (
+        cli(
+            [
+                "validate_spec",
+                "-s",
+                "in_ethernet.rflx",
+                "-m",
+                "Ethernet::Frame",
+                "-v",
+                f"{tmpdir}",
+            ]
+        )
+        == f"{subdir} is not a regular file"
+    )
+
+
+def test_cli_invalid_identifier() -> None:
+    assert (
+        cli(
+            [
+                "validate_spec",
+                "-s",
+                "in_ethernet.rflx",
+                "-m",
+                "Ethernet.Frame",
+                "-v",
+                "tests/data/ethernet/valid",
+                "-i",
+                "tests/data/ethernet/invalid",
+            ]
+        )
+        == 'invalid identifier "Ethernet.Frame" : id: error: "." in identifier parts'
```

Comment:
The error message looks strange with that `id: error:` in the middle. Do you have an idea to improve that?

Reply:
Will be fixed in #28.